examples/purchaseTesterTypescript/app/screens/FooterPaywallScreen.tsx

```diff
@@ -1,24 +1,22 @@
 import React from 'react';
-import { PaywallFooterView } from 'react-native-purchases-ui';
+import {PaywallFooterView} from 'react-native-purchases-ui';
 
-import { Text, View } from 'react-native';
-import { NativeStackScreenProps } from "@react-navigation/native-stack";
-import RootStackParamList from "../RootStackParamList";
+import {StyleSheet, View} from 'react-native';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import RootStackParamList from '../RootStackParamList';
 
 type Props = NativeStackScreenProps<RootStackParamList, 'FooterPaywall'>;
 
 const FooterPaywallScreen: React.FC<Props> = () => {
+  const styles = StyleSheet.create({
+    flex1: {
+      flex: 1,
+    },
+  });
+
   return (
-    <View style={{
-      justifyContent: 'space-between',
-      flexDirection: 'column',
-      flexGrow: 1,
-      backgroundColor: '#4b72f6',
-    }}>
-      <View style={{ backgroundColor: '#ff0000'}}>
-        <Text>Top</Text>
-      </View>
-      <PaywallFooterView/>
+    <View style={styles.flex1}>
+      <PaywallFooterView style={styles.flex1} />
```

Comment:
Ultimately, though, I think this is what we should vend as a component - the native view wrapped in a view, which we control. 

This way we can set the flex rules ourselves, otherwise we would have to rely on customers setting this, since we can't control the react-native component otherwise. 

Let me know if this isn't clear

Reply:
I see... so we offer a `PaywallFooterView` that is a `View` wrapping `InternalPaywallFooterView`. `InternalPaywallFooterView` always has flex:1 so it takes the whole space. Then devs can set their own flex rules on the `PaywallFooterView`, which will be the container of our `InternalPaywallFooterView`.

But... you're saying that if you don't set `flex:1` in the container, it doesn't detect the taps? How can we guarantee that devs set some size. I am not sure we can hardcode `flex: 1` in the container view, since that will take the whole view screen where `PaywallFooterView` is added which might have other views in it?

Reply:
I think it works just fine as long as _our_ container has flex:1. It feels like RN just got mixed up because we set a size in native land but no size rules in JS/CSS

Reply:
we should test it further though

Reply:
I've been testing this:
```
const PaywallFooterView = requireNativeComponent('RCPaywallFooterView');

export const FlexPaywallFooterView: React.FC<{ style?: StyleProp<ViewStyle> }> = ({style, ...props}) => (
  <View style={[{flex: 1}, style]}>
    <PaywallFooterView style={{flex: 1}}  {...props} />
  </View>
);
```

used like

```
const FooterPaywallScreen: React.FC<Props> = () => {
  const styles = StyleSheet.create({
    flex1: {
      flex: 1,
    },
  });

  return (
    <View style={styles.flex1}>
      <FlexPaywallFooterView/>
    </View>
  );
};
```

And it works fine. 

But now I've realized an issue that might change the way this works. The content in the corners is not correctly rendered and we need to make the `PaywallFooterView` a wrapper that contains the content of the screen, so it renders properly on top of the rest of the screen and the corner looks good

![image](https://github.com/RevenueCat/react-native-purchases/assets/664544/c6499dc0-02a9-41d7-9a07-02ac9b955155)

Reply:
Actually, that code sometimes adds some space at the bottom of the screen, I am not sure why. Anyways, I am going to look into the rounded corners